flocker/node/agents/test/test_blockdevice.py

```diff
@@ -844,6 +844,15 @@ class IBlockDeviceAPITestsMixin(object):
     """
     Tests to perform on ``IBlockDeviceAPI`` providers.
     """
+    def tearDown(self):
+        """
+        Destroy volumes created for the cluster used by the test.
+        """
+        volumes = self.api.list_volumes()
+
+        for volume in volumes:
+            self.api.destroy_volume(volume.blockdevice_id)
```

Comment:
It's not in the implementation.  It's an interaction with the loopback setup code in the test suite.

Does the EBS cleanup have to work this way or can it work more like the loopback cleanup works?  If the former, we may need to change how loopback cleanup is done.  If the latter, I'd just as soon convert this EBS cleanup code to work how the loopback cleanup works.